src/lib/operators/jit_operator/operators/abstract_jittable.hpp

```diff
@@ -21,6 +21,11 @@ class AbstractJittable {
 
   std::shared_ptr<AbstractJittable> next_operator() { return _next_operator; }
 
+  /**
+   * Update the operator according to the input table before specialization
```

Comment:
This doesn't tell me anything. Also, for a general "before" method, it takes quite specific arguments.